core/src/main/scala/kafka/server/metadata/BrokerMetadataPublisher.scala

```diff
@@ -232,6 +240,23 @@ class BrokerMetadataPublisher(
       if (_firstPublish) {
         finishInitializingReplicaManager()
       }
+
+      if (delta.featuresDelta != null) {
+        try {
+          val newFinalizedFeatures = new FinalizedFeatures(newImage.features.metadataVersionOrThrow, newImage.features.finalizedVersions, newImage.provenance.lastContainedOffset)
+          // Share version feature has been toggled.
+          if (!(newFinalizedFeatures.finalizedFeatures().getOrDefault(ShareVersion.FEATURE_NAME, 0.toShort) == finalizedShareVersion)) {
+            finalizedShareVersion = newFinalizedFeatures.finalizedFeatures().getOrDefault(ShareVersion.FEATURE_NAME, 0.toShort)
+            val shareVersion: ShareVersion = ShareVersion.fromFeatureLevel(finalizedShareVersion)
+            info(s"Share version has been toggled to $shareVersion")
```

Comment:
The message when the feature has been enabled is `Share version has been toggled to SV_1`. The string `SV_1` is only meaningful in the code. It would be better to say something like `Feature share.version has been updated to version 1.`